src/supervisor/watchers/handlers/cron-job.ts

```diff
@@ -117,6 +118,7 @@ export async function cronJobWatchHandler(
           .filter((container) => container.image !== undefined)
           .map((container) => container.image!),
       }),
+      deleteWorkloadFromScanQueue(workloadAlreadyScanned),
```

Comment:
can `deleteWorkloadFromScanQueue` be used somewhere after we are calling this function? Or because there is a different process for different types of workloads we need to have this step inside each function?

Reply:
The handlers are the entrypoints, there isn't a generic function at the moment where we can simply put this code to allow re-use. I know it can be a bit error prone! I think there is a way to generalize it but I wonder if it's a bit too premature